.github/workflows/pull-request-snapshot-release.yml

```diff
@@ -19,20 +19,22 @@ jobs:
         timeout-minutes: 30
 
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
               with:
                   ref: ${{ github.head_ref }}
 
             - uses: ./.github/actions/setup
 
             - name: Release with Gradle
               run: |
-                  BRANCH_NAME=PR-${{ env.PR_NUMBER }}
-                  sed -i '/version=/ s/-SNAPSHOT/-'"$BRANCH_NAME"'-SNAPSHOT/' ./gradle.properties
-                  ./gradlew build publishAllVersions -Pzowe.deploy.username=$ARTIFACTORY_USERNAME -Pzowe.deploy.password=$ARTIFACTORY_PASSWORD -Partifactory_user=$ARTIFACTORY_USERNAME -Partifactory_password=$ARTIFACTORY_USERNAME -PpullRequest=$BRANCH_NAME
+                  PR_NUMBER=PR-${{ env.PR_NUMBER }}
```

Comment:
The changes in this file are intentional?

Reply:
it is just to align the naming with v2 version